backend/src/main/java/ch/ase21/backend/service/NeighbourhoodService.java

```diff
@@ -78,9 +78,9 @@ private NeighbourhoodService() {/* void */}
     double breakEven =  averagePricePerApartment / (averageRevenuePerNight * 365);
     double score = 7 / breakEven; // Score of 1.0 if break even is in 7 years or less
     if(score > 1.0){
```

Comment:
Shouldn't this be `if(score > 10.0)` now?

Reply:
nope the score is scaled afterwards